raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -0,0 +1,471 @@
+
+Regulations regarding registration of pesticides and toxic substances
+(100) The Minister shall, in consultation with the relevant lead agencies, make regulations prescribing the contents of any application and the conditions for the registration of pesticides and toxic substances under this Act.
+
+Standards for noise
+(101) The Standards and Enforcement Review Committee shall, in consultation with the relevant lead agencies: [a] recommend to the Authority minimum standards for emissions of noise and vibration pollution into the environment as are necessary to preserve and maintain public health and the environment; [b] establish and submit to the Authority criteria and procedures for the measurement of noise and vibration pollution into the environment; [c] establish and submit to the Authority criteria and procedures for the measurement of subsonic vibrations; [d] establish and submit to the Authority standards for the emission of sub-sonic vibrations which are likely to have a significant impact on the environment; [e] recommend to the Authority guidelines for the minimisation of subsonic vibrations, referred to in paragraph [d] from existing and future sources; [f] establish and submit to the Authority noise level and noise emission standards applicable to construction sites, plants, machinery, motor vehicles, aircraft including sonic bonus, industrial and commercial activities; [g] recommend to the Authority measures necessary to ensure the abatement and control of noise from sources referred to in paragraph [f]; [h] measure the levels of noise emanating from the sources referred to in paragraph [f] details of which measurements shall be given to the owner or occupier of the premises from which the measurement was taken; and [i] recommend to the Authority guidelines for the abatement of unreasonable noise and vibration pollution emitted into the environment from any source.
+
+Noise in excess of established standards prohibited
+(102) Subject to the provisions of the Civil Aviation Act (Cap. 394), any person who emits noise in excess of the noise emission standards established under this Part commits an offence.
+
+Exemptions in respect of noise standards
+(103) [1] Notwithstanding the provisions of section 102, the Authority may on request grant a temporary permit not exceeding three months, allowing emission of noise in excess of established standards for such activities as fireworks, demolitions, firing ranges and specific heavy industry on such terms and conditions as the Authority may determine. [2] Where an exemption has been granted under subsection, [1], workers exposed to excessive levels of noise shall be adequately protected in accordance with the directives issued by the Authority.
+
+(104) Repealed by Act No. 5 of 2007, s. 78.
+
+(105) Repealed by Act No. 5 of 2007, s. 79.
+
+(106) Repealed by Act No. 5 of 2007, s. 80.(106) 
```

Comment:
The version of the legislation currently available from FAOLEX includes articles 104, 105 and 106.